cmd/influx/telegrafs.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/influxdata/influx-cli/v2/clients/telegrafs"
+	"github.com/influxdata/influx-cli/v2/pkg/cli/middleware"
+	"github.com/urfave/cli/v2"
+)
+
+func newTelegrafsCommand() *cli.Command {
+	var params telegrafs.ListParams
+	flags := append(commonFlags(), getOrgFlags(&params.OrgParams)...)
+	flags = append(flags, &cli.StringFlag{
+		Name:        "id",
+		Usage:       "Telegraf configuration ID to retrieve",
+		Aliases:     []string{"i"},
+		Destination: &params.Id,
+	})
+	return &cli.Command{
+		Name:  "telegrafs",
+		Usage: "List Telegraf configuration(s). Subcommands manage Telegraf configurations.",
+		Subcommands: []*cli.Command{
+			newCreateTelegrafCmd(),
+			newRemoveTelegrafCmd(),
+			newUpdateTelegrafCmd(),
+		},
+		Flags: flags,
+		Before: middleware.WithBeforeFns(withCli(), withApi(true)),
+		Action: func(ctx *cli.Context) error {
+			api := getAPI(ctx)
+			client := telegrafs.Client{
+				CLI:          getCLI(ctx),
+				TelegrafsApi: api.TelegrafsApi,
+			}
+			return client.List(ctx.Context, &params)
+		},
+	}
+}
+
+func newCreateTelegrafCmd() *cli.Command {
+	var params telegrafs.CreateParams
+	flags := append(commonFlags(), getOrgFlags(&params.OrgParams)...)
+	flags = append(flags, []cli.Flag{
+		&cli.StringFlag{
+			Name:        "description",
+			Usage:       "Description for Telegraf configuration",
+			Aliases:     []string{"d"},
+			Destination: &params.Desc,
+		},
+		&cli.StringFlag{
+			Name:        "file",
+			Usage:       "Path to Telegraf configuration",
+			Aliases:     []string{"f"},
+			Destination: &params.File,
+		},
+		&cli.StringFlag{
+			Name:        "name",
+			Usage:       "Name of Telegraf configuration",
+			Aliases:     []string{"n"},
+			Destination: &params.Name,
+		},
+	}...)
+	return &cli.Command{
+		Name:   "create",
+		Usage:  "The telegrafs create command creates a new Telegraf configuration.",
+		Flags:  flags,
+		Before: middleware.WithBeforeFns(withCli(), withApi(true)),
+		Action: func(ctx *cli.Context) error {
+			api := getAPI(ctx)
+			client := telegrafs.Client{
+				CLI:          getCLI(ctx),
+				TelegrafsApi: api.TelegrafsApi,
+			}
+			return client.Create(ctx.Context, &params)
+		},
+	}
+}
+
+func newRemoveTelegrafCmd() *cli.Command {
+	var params telegrafs.RemoveParams
+	flags := commonFlags()
```

Comment:
Ah, looks like the unused `OrgParams` from the client aren't even bound here.

For backwards-compatibility I'd add some minimal flags to the list you `append` below:
```go
{
    Name: "org",
    Hidden: true,
},
{
    Name: "org-id",
    Hidden: true,
}
```
That'll prevent the CLI from erroring if the flags are passed, but also remove them from the help-text so new users stop passing them.